internal/sarif/sarif.go

```diff
@@ -36,12 +37,24 @@ func Create(kubeauditReport *kubeaudit.Report) (*sarif.Report, error) {
 
 		auditor := strings.ToLower(result.Auditor)
 
+		formattedMap := make(map[string]string)
+
+		for k, v := range result.Metadata {
+			formattedMap[k] = "`" + v + "`"
```

Comment:
Question: are you sure that `v` won't contain a backquote? That could break your formatting. Example:

`meow ` meow`


If you are formatting for GitHub markdown, I think double-backticks may work - ``meow ` meow``
If you are formatting for humans, maybe ``strings.Replace(v, "`", "'")`` to sanitize? Humans will probably still get the gist.

If your formatting for HTML, maybe ``strings.Replace(v, "`", "&#96;")``? That is weird - it only came up because https://github.com/github/markup/issues/363

Reply:
usually, `v` is the name of the resource. I don't think it'll contain a backquote, but if anything changes down the line, we might run into issues. great point!

Yeah, I'm thinking Github markdown for sure. Considering the backticks for extra emphasis, but in the end we don't really need it? If I come to think of it, "`web`" looks weird...

Reply:
<img width="924" alt="Screenshot 2023-01-23 at 5 45 14 PM" src="https://user-images.githubusercontent.com/22174780/214166865-2d051787-fff5-409a-bde9-b036718ee6da.png">

Reply:
^ opted for this format